.travis.yml

```diff
@@ -1,4 +1,6 @@
+sudo: false
 language: node_js
 node_js:
-  - 0.10
+  - 0.12
+  - '0.10'
```

Comment:
`nvm` installs correctly, but travis itself reports it as `0.1` unless it's a string